src/main/webapp/components/LoadingWrapper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Let's handle hide and show in this component to decouple from App.js. App.js might be removed in that case.

Reply:
:+1: I decided to keep App to construct a typical react structure so we can later add in here both header and footer here.